andela/anbu-cfh
# Feature/#134249293/redesign gaming screen ui

#### What does this PR do?
Redesign the gaming UI

#### Description of Task to be completed?
Redesign the gaming user interface as well as fix minor issue with the authentication forms.

#### How should this be manually tested?
Run the app using `npm start`  then start a new game to see the redesigned interface.

#### Any background context you want to provide?
N / A

#### What are the relevant pivotal tracker stories?
#134249293 redesign the actual gaming ui.

#### Screenshots (if appropriate)
![screen shot 2016-12-14 at 11 10 59 am](https://cloud.githubusercontent.com/assets/23452643/21178045/0d2e313c-c1ee-11e6-8539-7b12089d7d8c.png)
![screen shot 2016-12-14 at 11 11 34 am](https://cloud.githubusercontent.com/assets/23452643/21178062/1e66aba0-c1ee-11e6-8f4f-60f8a602af39.png)

#### Questions:
N/A
